lib/benchee/formatters/units.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Benchee.Units do
+  @moduledoc """
+  Provides scaling and labeling functions for numbers. Use "count" functions
+  for numbers that represent countable things, such as iterations per second.
+  Use "duration" functions for numbers that represent durations, such as run
+  times.
+  """
+
+  @one_billion 1_000_000_000
+  @one_million 1_000_000
+  @one_thousand 1_000
+
+  def scale_count(count) when count >= @one_billion,  do: {count / @one_billion, :billion}
+  def scale_count(count) when count >= @one_million,  do: {count / @one_million, :million}
+  def scale_count(count) when count >= @one_thousand, do: {count / @one_thousand, :thousand}
+  def scale_count(count), do: {count, :one}
+
+  @spec format_count(number) :: String.t
```

Comment:
Ha, was thinking about adding type specs (actually there should be a ticket somewhere...). Wasn't too happy about my first experience with dialyxir and co. if you wanna do some work there, doing it in another PR sounds cool :)